server/odc-server/src/main/java/com/oceanbase/odc/server/web/controller/v1/DBRecyclebinController.java

```diff
@@ -59,26 +58,25 @@ public OdcResult<List<DBRecycleObject>> list(@PathVariable String sid) {
         return OdcResult.ok(recyclebinService.list(sessionService.nullSafeGet(SidUtils.getSessionId(sid))));
     }
 
-    @ApiOperation(value = "getDeleteSql", notes = "获取清除回收站指定对象的sql，sid示例：sid:1000-1:d:db1")
-    @RequestMapping(value = "/getDeleteSql/{sid:.*}", method = RequestMethod.PATCH)
-    public OdcResult<ResourceSql> getDeleteSql(@PathVariable String sid,
-            @RequestBody List<DBRecycleObject> resource) {
-        ConnectionSession session = sessionService.nullSafeGet(SidUtils.getSessionId(sid));
-        return OdcResult.ok(ResourceSql.ofSql(recyclebinService.getPurgeSql(session, resource)));
+    @ApiOperation(value = "purgeObject", notes = "purge a specific db object")
+    @RequestMapping(value = "/purge/{sid:.*}", method = RequestMethod.PATCH)
```

Comment:
if the API was totally redesigned, how about keep the v1 API , and use v2 API for new design?

we should keep API compatibility as  concept of changing.

Reply:
done, make v2 api and return SuccessResponse instead of OdcResult